model/simulationtests/output_objects.py

```diff
@@ -88,5 +88,13 @@
 # output_table.getSummaryReport(1).get == "AdaptiveComfortSummary"
 # output_table.summaryReportIndex("AdaptiveComfortSummary").get == 1
 
+###############################################################################
+#                       OUTPUTCONTROL:RESILIENCESUMMARIES                     #
+###############################################################################
+if openstudio.VersionString(openstudio.openStudioVersion()) > openstudio.VersionString("3.9.0"):
+    output_table.addSummaryReport("ThermalResilienceSummary")
+    output_rs = model.getOutputControlResilienceSummaries()
+    output_rs.setHeatIndexAlgorithm("Extended")  # Default is 'Simplified'
```

Comment:
Does "Simplified" give something "new" in the summary report, or is that just the option to maintain the old behavior?

Reply:
Simplified gives the same as not having the object at all in your IDF. (= old)